jackal_base/src/jackal_diagnostic_updater.cpp

```diff
@@ -227,6 +243,62 @@ void JackalDiagnosticUpdater::navsatCallback(const nmea_msgs::Sentence::ConstPtr
   }
 }
 
+#if defined(_WIN32)
```

Comment:
Any reason for not folding this into the function?

Reply:
Let me re-evaluate this. I think having Wifi and networking broken out into a separate ROS package would make it available to other solutions as well. Maybe that would be a longer term fix?

Reply:
I collapsed it as suggested

Reply:
Thanks!  That would be a good long term fix.  I will accept this to get it merged!